how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to present CV with Neptune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "THIzFFk81nme"
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "When training models with cross-validation, you can use Neptune namespaces to organize, visualize and compare models.\n",
+    "\n",
+    "By the end of this guide, you will learn how to organize your run to track cross-validation metadata, so that you can easily analyze the results.\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/showroom/e/SHOW-3700)\n",
+    "\n",
+    "[![image](https://files.gitbook.com/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-MT0sYKbymfLAAtTq4-t%2Fuploads%2FLRq4l78xEp31FFS5WzFM%2Fimage.png?alt=media&token=be868c83-7b1b-4314-b138-677531720f0e)](https://app.neptune.ai/o/common/org/showroom/e/SHOW-3700)\n",
```

Comment:
This screenshot is almost empty -> too much of a whitespace. Pls adapt it + pls make screenshots with zoomed in view.